src/components/UI/WalletButton/WalletButton.js

```diff
@@ -27,42 +25,19 @@ WalletButton.propTypes = {
 };
 
 const AccountWalletButton = ({account, chain, logoPath, onClick}) => {
-  const {breakpoint} = useBreakpoint(Breakpoint);
   const {colorOrangeSoda, colorWhiteOp10, colorWhite, colorDarkBlueGray} = useColors();
   const {accountWalletBtnTxt} = useHeaderTranslation();
 
-  const WALLET_LOGO_SIZE = 20;
-  const WALLET_LOGO_SIZE_MOBILE = 40;
-
-  const getText = () => {
-    const address = shortenAddress(account);
-    if (isMobileOrTablet(breakpoint)) {
-      return '';
-    } else if (!isDesktop(breakpoint)) {
-      return address;
-    }
-    return evaluate(accountWalletBtnTxt, {address});
-  };
-
-  const renderChainLabel = () => {
-    return isDesktop(breakpoint) ? <ChainLabel chain={chain} /> : null;
-  };
-
-  const renderWalletLogo = () => {
-    const logoSize = isMobile(breakpoint) ? WALLET_LOGO_SIZE_MOBILE : WALLET_LOGO_SIZE;
-    return <DynamicIcon path={logoPath} size={logoSize} />;
-  };
-
   return (
     <Button
-      className={toClasses(styles.accountWalletButton, styles[breakpoint.toLowerCase()])}
+      className={styles.accountWalletButton}
       colorBackground={colorWhiteOp10}
       colorBackgroundHover={colorDarkBlueGray}
       colorBorder={colorOrangeSoda}
       colorText={colorWhite}
-      iconLeft={renderWalletLogo()}
-      iconRight={renderChainLabel()}
-      text={getText()}
+      iconLeft={<DynamicIcon path={logoPath} size={20} />}
```

Comment:
maybe size 22 it's also ok so u can use `CircleLogoSize.XS`

Reply:
beside that LGTM

Reply:
I don't want `WalletButton` will be depends on `CircleLogo`. It doesn't make sense

Reply:
seems like we have to hold `IconSize` enum then

Reply:
done